### ISSUE TITLE
Description of dynamic_template_data is missing from the table for the send endpoint

### ISSUE THREAD

**Issue by f4lco** [sendgrid/docs#6305]
I assume the documentation for the send API is at least missing a parameter.

## Expected content

The table listing the parameters for the send email endpoint documentation should contain the parameter `dynamic_template_data`.

## Actual content

- The endpoint documentation (I'm referring to the paragraph below the "Handlebars" headline https://sendgrid.api-docs.io/v3.0/mail-send/v3-mail-send#handlebars) contains an example using `dynamic_template_data`, which is the correct name for this parameter.
- However, the parameter table below the example (the 'Definition' tab) lacks `dynamic_template_data`.
- However, the parameter table contains a parameter named `substitutions`, whose description sounds like it should describe `dynamic_template_data`. I come to the conclusion that maybe `substitutions` has been renamed to `dynamic_template_data`, but this is not reflected in all places. Description:

    > A collection of key/value pairs following the pattern "substitution_tag":"value to substitute". The key/value pairs must be strings. These substitutions will apply to the text and html content of the body of your email, in addition to the subject and reply-to parameters. The total collective size of your substitutions may not exceed 10,000 bytes per personalization object.

- Finally, when I make a request using `substitutions` (a parameter that the table says is usable with the send endpoint), the request comes back with 400 Bad Request.

To summarize, I think that the table contains a parameter which seemingly does not exist any longer (`substitutions`), and the table misses `dynamic_template_data`.  I suspect the former has renamed to the latter. In the worst case these are two distinct issues and might not be related.

**Link to page**: https://sendgrid.api-docs.io/v3.0/mail-send/v3-mail-send

**Comment by astuteape** [comment]
Hi @f4lco ,

Thank you for bringing this to our attention. The `dynamic_template_data` field was missing from the API reference. We have added it in.

We also made some notes in the field descriptions for `dynamic_template_data` and `substitutions` to clarify the difference between them. In short, `dynamic_template_data` is used to pass dynamic data to a Dynamic Transactional Template. You shouldn't use `substitutions` with a Dynamic Transactional Template. You can use `substitutions` to pass dynamic data for substitutions you wish to do when not using the SendGrid Dynamic Transactional Template system.

We hope this helps clarify, and thanks again for bringing it to our attention.

Best,
Twilio SendGrid Docs Team

**Comment by f4lco** [comment]
Thanks @astuteape !
